docs/ml-skills.md

```diff
@@ -48,17 +48,58 @@ This diagram shows the communication between the two AEAs.
         deactivate Search
         deactivate Ledger
 
-</div>  
+</div>
+<br>
 
-## Preparation instructions
+## Option 1: AEA Manager approach
 
-### Dependencies
+Follow this approach when using the AEA Manager Desktop app. Otherwise, skip and follow the CLI approach below. 
+
+### Preparation instructions
+
+Install the <a href="https://aea-manager.fetch.ai" target="_blank">AEA Manager</a>.
+
+### Demo instructions
+
+The following steps assume you have launched the AEA Manager Desktop app.
+
+1. Add a new AEA called `ml_data_provider` with public id `fetchai/ml_data_provider:0.28.0`.
+
+2. Add another new AEA called `ml_model_trainer` with public id `fetchai/ml_model_trainer:0.29.0`.
+
+3. Copy the address from the `ml_model_trainer` into your clip board. Then go to the <a href="https://explore-agent-land.fetch.ai" target="_blank">AgentLand block explorer</a> and request some test tokens via `Get Funds`.
+
+4. Run the `ml_data_provider` AEA. Navigate to its logs and copy the multiaddress displayed.
+
+5. Navigate to the settings of the `ml_model_trainer` and under `components > connections/fetchai/p2p_libp2p:0.22.0` update as follows (make sure to replace the placeholder with the multiaddress):
```

Comment:
@5A11 please change `components > connections/fetchai/p2p_libp2p:0.22.0` to `components > connections` with `fetchai/p2p_libp2p:0.22.0` or something similar where public id and extra path are separate code snippets. Otherwise our bump scripts won't work.

Reply:
I see, ok